bindings/java/src/org/sleuthkit/datamodel/OsAccountManager.java

```diff
@@ -414,7 +429,7 @@ public OsAccount newLocalLinuxOsAccount(String uid, String loginName, Host refer
 	/**
 	 * Creates a OS account with the given uid, name, and realm.
 	 *
-	 * @param uniqueId      Account sid/uid. May be null.
+	 * @param uniqueId      Account sid/uid. May be null. Saved in uppercase.
 	 * @param loginName     Login name. May be null only if SID is not null.
```

Comment:
Maybe update the comments for name/realm as well that they are saved in lower case?

Reply:
I was also thinking that, but it doesn't look like this method does the conversion, so maybe the comments should all say what case is expected?